base/src/org/spin/queue/util/QueueManager.java

```diff
@@ -136,7 +139,7 @@ public final QueueManager withContext(Properties context) {
 	/**
 	 * @param organizationId the tableId to set
 	 */
-	public final QueueManager withOrganizationId(int organizationId) {
+	private final QueueManager withOrganizationId(int organizationId) {
 		this.organizationId = organizationId;
```

Comment:
Validate, only set the organizationId field if there is no established RecordID.
If a RecordID exists, the organization ID of the PO takes precedence over that of the method.